packages/terra-application/package.json

```
@@ -57,7 +57,8 @@
     "lodash.debounce": "^4.0.8",
     "mutationobserver-shim": "^0.3.3",
     "prop-types": "^15.5.8",
-    "react-onclickoutside": "^6.7.1",
+    "react-draggable": "^4.4.3",
+    "react-onclickoutside": ">=6.7.1 <6.11.0",
```

Reverting on click outside change.
```suggestion
    "react-onclickoutside": "^6.7.1",
```